 kodilla-stream/src/main/java/com/kodilla/stream/array/AvarageCalculator.java

```diff
@@ -0,0 +1,18 @@
+package com.kodilla.stream.array;
+
+import java.util.stream.IntStream;
+
+public class AvarageCalculator implements ArrayOperations {
+
+    public void getAverage(int[] numbers) {
+        IntStream.range(0, numbers.length)
```

Comment:
Można wykorzystać metodę klasy Arrays. Jak w idei wpiszesz po zmiennej będącej tablicą ".streamy" to ci idea podpowie, jak z tablicy zrobić strumień :-)